### ISSUE TITLE
IRIS Q&As: Education -- Coverage

### ISSUE THREAD

**Issue by lwalsh617** [department-of-veterans-affairs/va.gov-team#17601]
**Issue**
For the launch of Resources and support MVP 1.1, content that fall into the "coverage" category under education & training needs to be developed based on the following:

- https://gibill.custhelp.va.gov/
- [Product guide pages ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/vfs-product-guides/Benefit.Hubs.Product.Guide.docx)
- Credible external sources as needed

Content will take the form of a Word document, but will be organized and labeled so that, when final it, it can be easily flowed into learning center page templates [currently in Drupal](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12066#issuecomment-675074504).

Content will meet [learning center template requirements](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/template-requirements.md).

Content will be tagged [per requirements](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/template-requirements.md#purpose-of-tags-in-the-learning-center). 

**Tasks**

- [x] SC to identify IRIS faqs to migrate (Laura)

- [x] SC to identify any content in the Contact Center that should inform editing of IRIS faqs, or be added as additional content (Laura)

- [x] SC to develop draft copy and review against content checklist  (Laura)

- [x] DEPO to review (Beth)

- [ ] SC to copyedit draft (Randi)

- [x] DEPO to get SME approval (Beth)

- [ ] SC to build Drupal page(s) (Selina, Randi or Laura)

- [ ] SC to proof Drupal page(s) (Randi)

- [x] DEPO to approve Drupal page(s) (Beth)

- [ ] SC to publish pages (Randi)

- [x] DEPO to alert SMEs that pages are live (Beth)

- [ ] SC to update R&S status document (Laura)

**Acceptance Criteria**

- [ ] Page live

**Up-to-date content here**

Please post most up-to-date content (whether copy deck, Drupal link, or live url) here rather than in comments below. 
[rs-1.1-education-coverage-v0.2.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5849796/rs-1.1-education-coverage-v0.2.docx)

**Comment by lwalsh617** [comment]
@bethpottsVADEPO -- The copy deck at the the top of this ticket is ready for your initial review. 
Thanks

**Comment by bethpottsVADEPO** [comment]
[rs-1.1-education-coverage.bp.9-11-21.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/7157651/rs-1.1-education-coverage.bp.9-11-21.docx)

Here's my feedback. See tracked changes, comments, and highlighting. Basically, instead of one coverage article, I'd like to break it up into a handful of articles using various templates. 

Some of the strategy work for you involves figuring out how to combine content from 2 Q&As into one article--probably using the About template, but I'm not sure. Let me know if you want to talk it through as you dig in.

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 Can you try revising the chart to this?:

**H1: How we determine your percent of Post-9/11 GI Bill benefits**

We base your award of benefits on how long you served on active duty and certain other factors. Keep reading on this page to find out how much you’re eligible for. Or go to your Post-9/11 GI Bill Statement of Benefits.

[Check your Statement of Benefits]

H2: Eligibility for 100% of Post 9/11 GI Bill benefits

You’re eligible for 100% of the full benefit if you meet any one of these requirements:

- You served on active duty and were awarded a Purple Heart on or after September 11, 2001, **or**
- You served on active duty for at least 30 continuous days and were discharged because of a service-connected disability, **or**
- You served on active duty for at least 36 months

H2: Eligibility for a percentage of benefits based on how long you served on active duty

If you served on active duty for less than 36 months, you’re not eligible for the full Post-9/11 GI Bill benefit. Find out what percentage of the full benefit you’re eligible for based on the amount of time you served on active duty:

- **Between 30 months and 36 months:** 90% of the full benefit
- **Between 24 months and 30 months:** 80% of the full benefit
- **Between 18 months and 24 months:** 70% of the full benefit
- **Between 6 months and 18 months:** 60% of the full benefit
- **Between 90 days and 6 months:** 50% of the full benefit

**Comment by lwalsh617** [comment]
@bethpottsVADEPO - Here's the updated page for your review: http://preview-prod.vfs.va.gov/preview?nodeId=35364

**Comment by bethpottsVADEPO** [comment]
Thank you, @lwalsh617 this is great! I just sent it off to SMEs for review.

**Comment by lwalsh617** [comment]
@bethpottsVADEPO and @RLHecht --  
Per discussion on Slack, the single Q&A article, **If I fail a class will I have to pay VA back?** is being eliminated.
Instead, it should appear in the existing R&S article [VA debt management]( https://www.va.gov/resources/va-debt-management/) > H2 **Understand your VA debt** > 4th accordion item **How can I avoid VA debt**? > as a **Note** under last bullet)
Per Beth: Beth to take care of clearing this with the SMEs and get that added to the page with Randi. 
Once this is done, can we delete the article I created? http://preview-prod.vfs.va.gov/preview?nodeId=35368
Thank you!!

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 Please go ahead and delete the article now. Also, open a new standard ticket for me so I can track the work I need to do for this. It won't go within the 4th accordion. It'll just be a note beneath the list of accordions there. Thanks for opening that ticket for me!

**Comment by lwalsh617** [comment]
@RLHecht -- Per Beth's comment ^ can you delete http://preview-prod.vfs.va.gov/preview?nodeId=35368 as I don't have the privileges to do so. (Then we can close this ticket.) Thanks!

@bethpottsVADEPO -- Here's the new ticket you asked for: https://github.com/department-of-veterans-affairs/va.gov-team/issues/30420

**Comment by RLHecht** [comment]
@lwalsh617 I just deleted the node above. I'll close the ticket!

**Comment by bethpottsVADEPO** [comment]
@RLHecht Reopening this ticket to pass this SME-reviewed article to you that can now get a final copyedit and you can set it to publish. Thanks!
http://preview-prod.vfs.va.gov/preview?nodeId=35364

**Comment by RLHecht** [comment]
This page is published. I removed the related link for "Tuition and fees the GI Bill covers" since that article is still in draft. @lwalsh617 We should create a ticket to add that link once that page is published so we don't lose track of it.

**Comment by lwalsh617** [comment]
@bethpottsVADEPO - Here's a link to our Slack discussion on 
http://preview-prod.vfs.va.gov/preview?nodeId=35367
http://preview-prod.vfs.va.gov/preview?nodeId=35366 
Net/net is that we decided to combined into one article: http://preview-prod.vfs.va.gov/preview?nodeId=35367 (Tuition and fees the GI Bill covers)

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 Here are substantive edits to incorporate into the new single article. I have a comment in the Word doc to you about trying to combine the examples in some way. Basically what I did is condense the content into 2 H2s about public and private school coverage, since the info about in-state and out-of-state coverage applies to public schools only. Now there's a need to rethink how many examples to provide and how to present the example(s). Would you work on that after you rearrange the article? Thanks!
[https://app.zenhub.com/files/133843125/84dbfe01-5b72-4512-8bb3-2a96c18db665/download](https://app.zenhub.com/files/133843125/84dbfe01-5b72-4512-8bb3-2a96c18db665/download)

**Comment by lwalsh617** [comment]
@bethpottsVADEPO -- I updated the single article per your copy deck above. I had to tighten up the H1 to meet the max char count. In incorporating the examples,  I added some H3s.  I also added 2 more links under related information. It's ready for you to review. Thanks!
<http://preview-prod.vfs.va.gov/preview?nodeId=35367>

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 Thanks, that works better with the H3s. I attached more edits here. To note: I removed "Post-9/11" everywhere because I think this applied to all GI Bill benefits. I'm going to verify this with SMEs. And we may need to change the H1 for the article we already published about the percentage of benefits (to remove Post-9/11). Will also ask them if that H1 should be broadened in this way. After you incorporate my edits, I'll send the preview to SMEs. Thanks!
[tuition_coverage.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/7274260/tuition_coverage.docx)

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 Would you note here if there are any other tickets in which we're tracking the above 3 articles? They aren't yet ready for Randi to copyedit.

**Comment by lwalsh617** [comment]
^ 35369 and 37440 are only tracked here.
37410 is also tracked on 30420 -- but that was for when we decided to eliminate as a single Q&A.

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 Here's how I'd like to proceed with these remaining articles related to this ticket (we should be able to wrap up this work in the upcoming sprint):
http://preview-prod.vfs.va.gov/preview?nodeId=35369 (in school part time)--pause on this one that has relatively low PVs in GIBHP (document in the github file that this one is paused)
http://preview-prod.vfs.va.gov/preview?nodeId=37410 (failing a class)--I edited this and it's ready to go to @RLHecht to copyedit, then I'll send it to SMEs (please document this in github file) 
http://preview-prod.vfs.va.gov/preview?nodeId=37440 (getting an extension)-- Edit this one to align with our plain language standards and our voice. Then please pass to @RLHecht to copyedit.

**Comment by RLHecht** [comment]
@lwalsh617 I copyedited the failing grade article and it's good to go to SMEs.

**Comment by RLHecht** [comment]
@laura I edited the GI Bill extension article as well. @bethpottsVADEPO these are ready for you now.

**Comment by bethpottsVADEPO** [comment]
@RLHecht To avoid the repetition in the H2s with "How do I request an extension if" let's pour this content into an About template instead of a multiple FAQ. This way the H2 can be: How to request an extension. And then H3s for the 3 different situations in which you can be eligible. 

I also edited throughout. So use my Word doc to create the new drupal node. I would like to try to avoid nested sub-bullets, so I reworked the text in that Q&A. See if you can chunk the text with bolding--it's a lot of text and a lot of bullets. I'm not sure if it makes more sense to bold the lead-in to the bullets or the line of text before that (or partial, first part of the line of text). I tried the latter, but would you play with it and see what works the best? And until we're 100% sure we're going with the About template version, keep the FAQ drupal node. Thanks!

[https://app.zenhub.com/files/133843125/fb9ebadf-9c7d-4135-87d0-741c4270f37f/download](https://app.zenhub.com/files/133843125/fb9ebadf-9c7d-4135-87d0-741c4270f37f/download)

**Comment by lwalsh617** [comment]
@bethpottsVADEPO - The Word doc  attached ^ is an old version of the article. Just making sure this is the document you meant to share as the bulk of the edits no longer apply. I'm looking through it to pull out anything I see that's still relevant, but let me know if there's a more current version. Thanks.

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 Yes, I had started editing simultaneously to your edit, so I combined our edits in this doc. Thank you.

**Comment by RLHecht** [comment]
@bethpottsVADEPO This is ready for you now. I went back and forth with the bolding too and ultimately bolded the setup line before the bullets. Let me know what you think. http://preview-prod.vfs.va.gov/preview?nodeId=41926

**Comment by bethpottsVADEPO** [comment]
@lwalsh617 @RLHecht Thank you for all this work! I have just one question:
"You served on active duty after the dates reported on your application for GI Bill benefits."

Does this mean you served on active duty after the date you applied for GI Bill benefits (or is that oversimplifying)?

**Comment by lwalsh617** [comment]
@bethpottsVADEPO I believe what you said above is correct. I haven't been able to find a source to corroborate it, though, so it's something we should flag for the SME to confirm.

**Comment by RLHecht** [comment]
Closing this ticket and publishing articles in ticket #34121
